### PR TITLE
1.4.0

### PR DESCRIPTION
**Pull request by TitusKirch**

### CODE REVIEW

**Comment by TitusKirch** [comment]
first fix #30

**Comment by TitusKirch** [comment]
> I haven't tested the changes, but the code looks good to me.
Do you have any idea right away what the cause of #30 is?

**Comment by Krymonota** [comment]
> Do you have any idea right away what the cause of #30 is?

Not really, sorry.

File: package.xml [hunk 1]
```diff
@@ -5,8 +5,8 @@
 		<packagename language="de">Simple FAQ</packagename>
 		<packagedescription>Add a FAQ to your WSC.</packagedescription>
 		<packagedescription language="de">Fügt ein FAQ in deinem WSC hinzu.</packagedescription>
-		<version>1.4.0 Alpha 1</version>
-		<date>2020-05-12</date>
+		<version>1.4.0</version>
+		<date>2020-06-15</date>
```
Comment (Hanashi) [inline package.xml:9]: hier aktuelles Datum einfügen

**Pull request merged**